src/azure-cli/azure/cli/command_modules/cosmosdb/aaz/latest/cosmosdb/postgres/cluster/_list_by_resource_group.py

```diff
@@ -0,0 +1,385 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cosmosdb postgres cluster list-by-resource-group",
```

Comment:
Usually we will combine `list-by-resource-group` with `list` by subscription. You can follow this doc [link](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#merge-similar-resource-urls-in-one-command)

Reply:
Combined list operations as suggested.